depends/packages/fontconfig.mk

```diff
@@ -11,6 +11,7 @@ define $(package)_set_vars
 endef
 
 define $(package)_config_cmds
+  export PKG_CONFIG_PATH=$(host_prefix)/lib/pkgconfig && \
```

Comment:
I remember now that I looked back at my commits why I did that. Doing a depends build for RasPi using the [RasPi toolchain](https://github.com/raspberrypi/tools) is what prompted me to add that, because I was getting pkgconfig related errors (couldn't find dependencies or something like that, don't have the errors offhand). As you can see from [this commit](https://github.com/josephbisch/BitcoinArmory/commit/873d52c6c804091e09da17574b385ecd3bb3aa5d), I had to add something similar to a number of packages.

It's a little hard to navigate the RasPi tools repo, so the tools I am specifically using are [these tools](https://github.com/raspberrypi/tools/tree/master/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/bin)